src/protocol.h

```diff
@@ -35,7 +35,7 @@ extern unsigned char pchMessageStart[4];
 class CMessageHeader
 {
     public:
-	static constexpr size_t MESSAGE_START_SIZE = 4;
+	    static constexpr size_t MESSAGE_START_SIZE = 4;
```

Comment:
The indentation of this line seems off.

Reply:
@Pythonix make sure your editor is set to spaces instead of tabs, and it is 4 spaces.